vhdbuilder/packer/install-dependencies.sh

```diff
@@ -77,12 +77,19 @@ else
   done
 fi
 
+mkdir -p /etc/systemd/system/chrony.service.d
+tee -a /etc/systemd/system/chrony.service.d/10-chrony-restarts.conf > /dev/null <<'EOF'
+Restart=always
```

Comment:
I think we need a `[Service]` directive at the top here

Reply:
also, we should probably make sure to only do this _after_ chrony has been installed and the unit has been setup, otherwise there's a risk of this configuration being clobbered - probably something good to test with agentbaker e2e's.

Reply:
Agree. Have added e2e tests for ubuntu 1804, 2204, azure linuxv2 and marinerv2. Now to get them passing...